docs/user-guide/cli-cicsplugin.md

```diff
@@ -41,27 +36,58 @@ Use one of the following methods to install or update the plug-in:
 
 ## Creating a user profile
 
-You create a cics profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
-- **Create plug-in profiles using a configuration file:** Specify your profile and connection details in the `zowe.config.json` configuration file.
-- **Create plug-in profiles using a command:** Issue the `zowe profiles create` command to create the profile.
-We recommend that you create profiles using the configuration file. We do not recommend using profile commands because we are removing them from a future major release.
+After you install the plug-in, create a CICS profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed.
+
+Specify your plug-in profile and connection details in the `zowe.config.json` configuration file.
 
 ### Creating plug-in profiles using a configuration file
 
-When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the CICS plug-in, the commands create an entry for a cics profile in your `zowe.config.json` file.
+When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the CICS plug-in and then issue a `zowe config` command **[correct?]**, the command creates an entry for a CICS profile in your `zowe.config.json` file.
 
 Alternatively, you can create a CICS profile manually by adding a section that contains the configuration details to your `zowe.config.json` configuration file.
 
-1. Browse to the following directory `C:\Users\<username>\.zowe`
+#### Creating a CICS profile with a command
+
+1.  Install the CICS Plug-in for Zowe CLI.
+2.  Create a CICS profile:
+
+    ```
+    zowe config init
+    ```
+3.  If using a non-standard port, set the port number to your CICS connection: **[is this step correct/applicable to cics?]**
```

Comment:
@adam-wolfe - please respond

Reply:
Looking at the code for the CICS plug-in it looks like the default value for port is 443 which is the same as z/OSMF, so I think the default would probably not work for most users. Plus I'm not sure that there is a known default port for the CICS CMCI server.

So, I think users will probably need to specify the port in all cases. @zFernand0 may be able to shed some light here.

Reply:
Sorry for the delay. I'm afraid that based on our internal development and some legacy configuration around the CMCI, we have had a default port on CICS of 1491 for a long time.
I believe that we would want to modify it to be port 0 so that people are forced to provide it in Zowe V3.